tools/storm/e2e/scenario/trident.go

```diff
@@ -4,32 +4,101 @@ import (
 	"fmt"
```

Comment:
a README.md file in this folder would help people understand how to run this locally

Reply:
Good idea. Added a (hopefully) comprehensive README under `tools/storm/e2e`